### ISSUE TITLE
How to destroy plugin instance?

### ISSUE THREAD

**Issue by sbespalov** [CarlosBonetti/jquery-loading#10]
Hi.
Is there any methods to destroy plugin instance? Something like `$( element ).loading('destroy')`?
The problem is that when i am doing `$( element ).loading('stop')` the overlay `div` element still exists in the DOM.

**Comment by CarlosBonetti** [comment]
Hi! Currently, there is no such method but I think it would be a nice addition! You can open a PR if you like.

For now, you can use:

```js
$(element).Loading().overlay.remove();
```

**Comment by sbespalov** [comment]
build failed

**Comment by sbespalov** [comment]
https://travis-ci.org/CarlosBonetti/jquery-loading/jobs/171896129

**Comment by CarlosBonetti** [comment]
Thanks for the PR! I think the error is caused by a bad grunt configuration... I will take a look as soon as I can

**Comment by CarlosBonetti** [comment]
Travis build is now passing after some dependencies update.

For reference, the overlay can now be destroyed using.

```js
$( selector ).loading('destroy');
```

Shoud be available at version 1.3.0.

Thanks for the help, @sbespalov !
